bears/installation_tool.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import importlib
+import re
+import subprocess
+import sys
+
+from coala_utils.Question import ask_question
+from coalib.collecting.Collectors import get_all_bears_names
+from coalib.collecting.Importers import iimport_objects
+from coalib.misc.Shell import call_without_output
+from prompt_toolkit import prompt
+from prompt_toolkit.contrib.completers import WordCompleter
+from pygments.lexers import SqlLexer
+
+
+def install_pip_package(package_name):
+    """
+    Uses call to install a PyPi package.
+
+    :param package_name: The package to be installed.
+    """
+    subprocess.call(['sudo', 'pip3', 'install', package_name, '--upgrade'])
```

Comment:
I'm not sure but can `pip3` be just `pip` under any circumstances? I guess `python3 -m pip` is a safer option.